src/escalus_session.erl

```diff
@@ -104,7 +104,11 @@ bind(Conn, Props) ->
         <<"SASL-ANON">> ->
             JID = exml_query:path(BindReply, [{element, <<"bind">>}, {element, <<"jid">>}, cdata]),
             TMPUsername = escalus_utils:get_username(JID),
-            lists:keyreplace(username, 1, Props, {username, TMPUsername});
+            Props1 = case lists:keymember(jid, 1, Props) of
+                       true -> lists:keyreplace(jid, 1, Props, {jid, escalus_utils:get_short_jid(JID)});
```

Comment:
Line 108 is too long:                        true -> lists:keyreplace(jid, 1, Props, {jid, escalus_utils:get_short_jid(JID)});.